src/validation.cpp

```diff
@@ -365,7 +365,9 @@ void CChainState::MaybeUpdateMempoolForReorg(
     // previously-confirmed transactions back to the mempool.
     // UpdateTransactionsFromBlock finds descendants of any transactions in
     // the disconnectpool that were added back and cleans up the mempool state.
-    m_mempool->UpdateTransactionsFromBlock(vHashUpdate);
+    uint64_t ancestor_limit = gArgs.GetArg("-limitancestorcount", DEFAULT_ANCESTOR_LIMIT);
```

Comment:
Maybe worth renaming `ancestor_limit` into `ancestor_limit_count`, to be more symmetrical with `ancestor_limit_size`/`limitancestorsize` pair?

Also, `UpdateTransactionsFromBlock` and `UpdateForDescendants` has these parameters phrased a bit differently with `ancestor_size_limit` and `ancestor_count_limit` ("limit" goes last). Maybe worth unifying all these new names?

Reply:
Also, `ancestor_limit` and `ancestor_limit_size` could be `const`.